src/envoy/mixer/utils.cc

```diff
@@ -54,8 +54,14 @@ bool GetSourceUser(const Network::Connection* connection, std::string* user) {
   if (connection) {
     Ssl::Connection* ssl = const_cast<Ssl::Connection*>(connection->ssl());
     if (ssl != nullptr) {
-      *user = ssl->uriSanPeerCertificate();
-      return true;
+      std::string result = ssl->uriSanPeerCertificate();
```

Comment:
Is there any test we need to update as well?

Reply:
I have run all existing tests in /proxy via bazel test ..., and all of them pass.